### PR TITLE
Fix setting due_payment in storage phase

### PR DESCRIPTION
**Pull request by EmelyanenkoK**

### CODE REVIEW

File: common/global-version.h [hunk 1]
```diff
@@ -19,6 +19,6 @@
 namespace ton {
 
 // See doc/GlobalVersions.md
-const int SUPPORTED_VERSION = 6;
```
Comment (Jimvilcaaa) [inline common/global-version.h:22]: 0

File: crypto/block/transaction.h [hunk 2]
```diff
@@ -82,6 +82,7 @@ struct StoragePhaseConfig {
   td::RefInt256 freeze_due_limit;
   td::RefInt256 delete_due_limit;
   bool enable_due_payment{false};
+  int global_version = 0;
```
Comment (Jimvilcaaa) [inline crypto/block/transaction.h:85]: 5

File: common/global-version.h (hunk 1 above)
Comment (Dymas7548) [reply common/global-version.h:22]: - [ ]

**Review by ahmadrashdy14 — COMMENTED** [review]
ahmadrashdy14@gmail.com

**Pull request merged**